docs/docs/sample-scripts/teams/list-all-tabs-teams/index.mdx

```diff
@@ -50,7 +50,6 @@ List all tabs in Microsoft Teams teams in the tenant and exports the results in
           ChannelMemberShipType = $channel.membershipType
           TabId                 = $tab.id
           TabNameDisplayName    = $tab.DisplayName
-          TeamsAppTabId         = $tab.teamsAppTabId
```

Comment:
Instead of removing this, let's reference to the `teamsApp.id` object.
Also let's update the metadata file and increase the version to v10.